helpers/deploymentConfig.ts

```diff
@@ -1,40 +1,87 @@
+import mainnetDeployments from "@venusprotocol/venus-protocol/networks/mainnet.json";
+import testnetDeployments from "@venusprotocol/venus-protocol/networks/testnet.json";
 import { Contract } from "ethers";
-import hre from "hardhat";
-import { DeployFunction } from "hardhat-deploy/dist/types";
-import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { ethers } from "hardhat";
 
-interface Feed {
+export interface Feed {
   [key: string]: string;
 }
 
-interface Config {
+export interface Config {
   [key: string]: Feed;
 }
 
-interface Asset {
+export interface Asset {
   token: string;
   address: string;
   oracle: string;
   price: string;
 }
 
-interface Assets {
+export interface Assets {
   [key: string]: Asset[];
 }
 
-interface Oracle {
+export interface NetworkAddress {
+  [key: string]: string;
+}
+
+export interface PreconfiguredAddresses {
+  [key: string]: NetworkAddress;
+}
+
+export interface AccessControlEntry {
+  caller: string;
+  target: string;
+  method: string;
+}
+
+export interface Oracle {
   oracles: [string, string, string];
   enableFlagsForOracles: [boolean, boolean, boolean];
   underlyingOracle: Contract;
   getTokenConfig?: (asset: Asset, networkName: string) => void;
+  getDirectPriceConfig?: (asset: Asset) => void;
   getStalePeriodConfig?: (asset: Asset) => string[];
 }
 
-interface Oracles {
+export interface Oracles {
   [key: string]: Oracle;
 }
 
-const chainlinkFeed: Config = {
+export const addr0000 = "0x0000000000000000000000000000000000000000";
+export const DEFAULT_STALE_PERIOD = 24 * 60 * 60; // 24 hrs
+export const ANY_CONTRACT = ethers.constants.AddressZero;
+
+export const ADDRESSES: PreconfiguredAddresses = {
+  bsctestnet: {
+    vBNBAddress: testnetDeployments.Contracts.vBNB,
+    WBNBAddress: testnetDeployments.Contracts.WBNB,
+    VAIAddress: testnetDeployments.Contracts.VAI,
+    pythOracleAddress: "0xd7308b14BF4008e7C7196eC35610B1427C5702EA",
+    sidRegistryAddress: "0xfFB52185b56603e0fd71De9de4F6f902f05EEA23",
+    acm: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
+    timelock: testnetDeployments.Contracts.Timelock,
+  },
+  bscmainnet: {
+    vBNBAddress: mainnetDeployments.Contracts.vBNB,
+    WBNBAddress: mainnetDeployments.Contracts.WBNB,
+    VAIAddress: mainnetDeployments.Contracts.VAI,
+    pythOracleAddress: "0x4D7E825f80bDf85e913E0DD2A2D54927e9dE1594",
```

Comment:
I would add these addresses for `pythOracleAddress`:
* ethereum: `0x4305fb66699c3b2702d4d05cf36551390a4c69c6`
* sepolia: `0x2880ab155794e7179c9ee2e38200202908c17b43`

(According to the [official Pyth documentation](https://docs.pyth.network/documentation/pythnet-price-feeds/evm))

Then I would deploy the PythOracle rerunning the [deploy script](https://github.com/VenusProtocol/oracle/pull/124/files#diff-a6e3e0aedc51ca991268c9403e7a26e9102952a4996186ded649a5478c866b02R110)